akka-osgi/build.sbt

```diff
@@ -1,6 +1,7 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, OSGi }
 
 AkkaBuild.defaultSettings
+AkkaBuild.dontPublishSettings
```

Comment:
the principle that `publishArtifact := false` => exclude from dist is correct though